packages/plugin-react-query/src/templates/reactQueryCommon.liquid

```diff
@@ -39,7 +39,12 @@ Omit<FetcherOptions<
   {%- endif %}
 }
 
-export function {{fetcherName}}(props: {{fetcherPropsName}}): Promise<{{okResponseName}}> {
+export interface {{fetcherPropsName}}ReturnValue {
+  content: {{okResponseName}}
+  headers: Record<string, any>
```

Comment:
do we have to use any?

Reply:
Tried changing it to `Record<string, unknown>` but ran into further issues from downstream libraries. Will check if it can be a quick fix.